File: Problem 7/problem7.py
print('''By listing the first six prime numbers: 2, 3, 5, 7, 11, 13, and we can see that the 6th prime is 13.

What is the 10001st prime number?''')

def is_prime(n):
    if n < 2:
        return False
    for j in range(2, int(n**0.5) + 1):
        if n % j == 0:
            return False
    return True

j = 0
i = 1

while j < 10001:
    i += 1
    if is_prime(i):
        j += 1

print("The answer is :", i)

File: Problem 7/problem7.c
#include <stdio.h>
#include <math.h>

int is_prime(int number);

int main(){

	int i = 1;
	int j = 0;

	while(j<10001){
		i+=1;
		if (is_prime(i)){
			j += 1;
		}
	}
	printf("answer: %d\n",i);
	return 0;
}

int is_prime(int number){
	int high = sqrt(number);
	for (int i = 2; i <= high; i++){
		if(!(number % i)){
		 	return 0;
		 }
	}
	return 1;
}
